sample/grpc-sample.yaml

```diff
@@ -8,14 +8,14 @@ spec:
     import:
       - ./pkg/server
     protofile: server.proto
-    serverReflection: false
+    serverReflection: true
```

Comment:
This part is missing from https://github.com/LinuxSuRen/api-testing/blob/c582610566e763d1d69fa5a4a01a302165b90578/pkg/server/server.proto#L83

Please don't forget running the following command after changing `server.proto`:

`make grpc grpc-gw`